multi_agent_research_assistant_ag2/research_assistant.py

```
@@ -0,0 +1,149 @@
+
+def run_research(topic: str) -> str:
+    llm_config = build_llm_config()
+
+    # ── Agents ─────────────────────────────────────────────────────────────────
+
+    researcher = AssistantAgent(
+        name="researcher",
+        system_message=f"""You are a research specialist. Your job is to gather
+comprehensive information about the given topic using web_search and fetch_page_content.
+Perform at least 3 searches and fetch content from 2+ pages.
+Summarise all findings clearly. End with: RESEARCH COMPLETE.""",
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)